chore(curriculum): remove wiki reference from cip (#46457)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/i-before-e-except-after-c.md

# --description--

> **<a href="http://www.rosettacode.org/wiki/I_before_E_except_after_C" target="_blank" rel="noopener noreferrer nofollow">"I before E, except after C"</a>** is used to aid with the English language spelling, using the rythem of the word. If one is unsure whether a word is spelled with the digraph `ei` or `ie`, the rhyme suggests that the correct order is `ie` unless the preceding letter is `c`, in which case it may be `ei`.

Reviewer: ```suggestion
**<a href="http://www.rosettacode.org/wiki/I_before_E_except_after_C" target="_blank" rel="noopener noreferrer nofollow">"I before E, except after C"</a>** is a general rule for English language spelling. If one is unsure whether a word is spelled with the digraph `ei` or `ie`, the rhyme suggests that the correct order is `ie` unless the preceding letter is `c`, in which case it may be `ei`.
```

Write a function that accepts a word and check if the word follows this rule. The function should return true if the word follows the rule and false if it does not.

> here are the words tested: receive, science, imperceivable, inconceivable, insufficient, and omniscient

Reviewer: This just repeats the information from the test suite. I don't think this is needed.

Reviewer: You don't need to visit an external site to read the list of tests that we're running?

Author: That is actually true🤣
